modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/QueryDescriptor.java

```diff
@@ -77,6 +84,7 @@ public class QueryDescriptor {
         this.skipReducerOnUpdate = skipReducerOnUpdate;
         this.batched = batched;
         this.qryInitiatorId = qryInitiatorId;
+        this.label = label;
```

Comment:
set nullable to not nullable

Reply:
added nullable to qryInitiatorId and label.